src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -246,6 +246,7 @@ export default class FileField extends React.Component {
                 id={`${idSchema.$id}_add_label`}
                 htmlFor={idSchema.$id}
                 className="usa-button usa-button-secondary"
+                aria-label={uiSchema['ui:title'] || schema.title}
               >
                 <span
```

Comment:
I'm not sure why a `<span>` is being used as a button here. Should this be updated @1Copenut?

Either way, I think it needs an `aria-label` as well.

Reply:
@Mottie We deliberately put the `<span>` in to accept the `role="button"`. Our label covers up the `input[type="file"]` so we needed to ensure the visual "button" announced itself and responded as a button. Adding the role smoothed it out for screen readers, but we had to add it to the span b/c labels can't have their implicit role removed.

We used this article as our guide: https://accessabilly.com/a-styled-accessible-file-upload/